packages/lua-language-server/build.sh

```diff
@@ -69,9 +67,9 @@ termux_step_make_install() {
 
 	chmod 744 "${TERMUX_PREFIX}/bin/${TERMUX_PKG_NAME}"
 
-	install -Dm744 -t "${INSTALL_DIR}"/bin/Android ./bin/Android/"${TERMUX_PKG_NAME}"
+	install -Dm744 -t "${INSTALL_DIR}"/bin ./bin/"${TERMUX_PKG_NAME}"
 	install -Dm644 -t "${INSTALL_DIR}" ./{main,debugger}.lua
-	install -Dm644 -t "${INSTALL_DIR}"/bin/Android ./bin/Android/main.lua
+	install -Dm644 -t "${INSTALL_DIR}"/bin ./bin/main.lua
```

Comment:
Maybe this should go to `share/` or somewhere similar ? `bin/` usually contains binaries ?

Reply:
Nah, consider checking out `INSTALL_DIR` environment variable. Also Arch is installing the file at that location only https://archlinux.org/packages/community/x86_64/lua-language-server/

:)

Reply:
No @thunder-coding  the arch package is installing the wrapper script in `bin/` and installing the `lua-language-server` binary and the `main.lua` file under `lib/` . See [here](https://github.com/archlinux/svntogit-community/blob/788278b1dca83ec6bcccaeeae6d8b06b758d25e1/trunk/PKGBUILD#L52-L57)